### PR TITLE
Ornamenting Jingle Bells

### PR DESCRIPTION
**Pull request by ology**
How about some music this year? :D

### CODE REVIEW

File: 2023/incoming/ornamenting-jingle-bells.pod [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        $counter = $counter == MAX ? 0 : $counter + 1;
+    }
+
+    # Write out the fancy score as a MIDI file
+    $melody->write_score("$0.mid");
+
+=head2 SYNOPSIS
+
+      $ perl jingle-bells
+      $ timidity jingle-bells.mid
```
Comment (briandfoy) [inline 2023/incoming/ornamenting-jingle-bells.pod:128]: Maybe a quick paragraph on playing MIDI files, with something that works on Windows too.

Comment (ology) [reply 2023/incoming/ornamenting-jingle-bells.pod:128]: ok

**Comment by oalders** [comment]
Closes #288

File: 2023/incoming/ornamenting-jingle-bells.pod (hunk 1 above)
Comment (ology) [reply 2023/incoming/ornamenting-jingle-bells.pod]: I have added instruction sections for playing MIDI on different systems.

**Review by oalders — APPROVED** [review]
\o/

**Comment by oalders** [comment]
It looks great! https://perladvent.org/2023/2023-12-01.html

Thank you @oalders and @briandfoy! If anything needs to be fixed, we can still do that.

**Pull request merged**